### PR TITLE
Improved hero banner design

### PR DESCRIPTION
**Pull request by juozasg**
Please merge this gradient if you can. It contains a CSS3 linear repeating gradient at 15 degree angle that matches your previous design idea. It also contains two radial gradients that gives a 3D look. I think it looks good but obviously that is subjective. 

I also added an hacky bash (sh) alert to hack/import-flux2-assets.sh that make sure the script will fail with correct warning if you do not have jq installed.

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+---
```
Comment (dholbach) [inline content/en/_index.md:1]: This PR seems to re-add `content/en/_index.md`. I moved over over to `content/en/_index.html` as we added a lot of HTML in the landing page and upon using a newer `docsy` version our generated HTML wasn't valid any more, etc. We can't have `_index.md` and `_index.html` for generating the same page.

**Comment by dholbach** [comment]
Thanks for all the other changes - this looks a lot cleaner. I like it very much! <3

**Review by dholbach — CHANGES_REQUESTED** [review]
If you remove `_index.md` we should be ready to merge this!

File: content/en/_index.md (hunk 1 above)
Comment (juozasg) [reply content/en/_index.md]: On juozasg:main branch I have deleted _index.md that I accidentally committed.

**Review by dholbach — APPROVED** [review]
Nice work @juozasg !

**Pull request merged**